### PR TITLE
[Issue 517]fixed group by count query

### PR DESCRIPTION
**Pull request by ashokblend**
Carbon is assuming below query as count query, which is not correct.
select ,count() from table group by having count()>1

Fix is made to do proper check for count query.

### CODE REVIEW

**Comment by ravipesala** [comment]
retest this please

**Comment by ravipesala** [comment]
Please add test case for this scenerio

File: integration/spark/src/main/scala/org/apache/spark/sql/CarbonOperators.scala [hunk 1]
```diff
@@ -534,6 +534,21 @@ case class CarbonCubeScan(
...
 
+  /**
+   * return true if query is count query
+   * @return
+   */
+  def isCountQuery: Boolean = {
+    if (buildCarbonPlan.isCountStarQuery() && null == buildCarbonPlan.getFilterExpression &&
+        buildCarbonPlan.getDimensions.size() < 1 && buildCarbonPlan.getMeasures.size() < 2 &&
+        buildCarbonPlan.getDimAggregatorInfos.size() < 1 &&
+        buildCarbonPlan.getExpressions.size() == 0) {
```
Comment (ravipesala) [inline integration/spark/src/main/scala/org/apache/spark/sql/CarbonOperators.scala]: I am doubtful about this check `buildCarbonPlan.getExpressions.size()`. No where we are adding expressions in our code.

**Comment by gvramana** [comment]
retest this please

**Pull request merged**